code/modules/surgery/organs/organ.dm

```diff
@@ -248,13 +248,18 @@ var/list/organ_cache = list()
 		return
 	switch (severity)
 		if (1.0)
-			take_damage(0,20)
+			take_damage(20,0)
```

Comment:
That's incorrect, @FalseIncarnate 

https://github.com/ParadiseSS13/Paradise/blob/0ac7c0d4cd6fc11abde56a2a49eb57b013034545/code/modules/surgery/organs/organ_external.dm#L162

While it's true for the base organ type, it's most certainly not true for external.